Polygon_mesh_processing/doc/Polygon_mesh_processing/Doxyfile.in

```diff
@@ -28,4 +28,6 @@ EXPAND_AS_DEFINED      = CGAL_PMP_NP_TEMPLATE_PARAMETERS \
 EXCLUDE = ${CGAL_PACKAGE_INCLUDE_DIR}/CGAL/Polygon_mesh_processing/internal
 EXCLUDE_SYMBOLS += experimental
 
-HTML_EXTRA_FILES           =  ${CGAL_PACKAGE_DOC_DIR}/fig/selfintersections.jpg
+HTML_EXTRA_FILES           = ${CGAL_PACKAGE_DOC_DIR}/fig/selfintersections.jpg \
+                             ${CGAL_PACKAGE_DOC_DIR}/fig/mesh_smoothing.jpg \
+                             ${CGAL_PACKAGE_DOC_DIR}/fig/shape_smoothing.jpg
```

Comment:
Something is wrong:
```
error: Extra file '/home/cgal-testsuite/cgal_doc_build/CGAL-5.0-Ic-146/doc/Polygon_mesh_processing/fig/mesh_smoothing.jpg' specified in HTML_EXTRA_FILES does not exist!
error: Extra file '/home/cgal-testsuite/cgal_doc_build/CGAL-5.0-Ic-146/doc/Polygon_mesh_processing/fig/shape_smoothing.jpg' specified in HTML_EXTRA_FILES does not exist!
```
https://cgal.geometryfactory.com/CGAL/Manual_doxygen_test/CGAL-5.0-Ic-146/logs2/Polygon_mesh_processing.log

Reply:
Fixed now; sorry about that.